eo-runtime/pom.xml

```diff
@@ -66,6 +66,14 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
```

Comment:
@nlchar no need to do this, 1.8 is the default version coming from the parent POM: https://github.com/jcabi/jcabi-parent/blob/master/pom.xml#L104-L105